packages/dune/dune.3.0.0/opam

```diff
@@ -33,6 +33,7 @@ conflicts: [
   "dune-release" {< "1.3.0"}
   "js_of_ocaml-compiler" {< "3.6.0"}
   "jbuilder" {= "transition"}
+  "base-effects"
```

Comment:
`base-effects` is a package that depends on a compiler with the "effect syntax" so everytime a project uses e.g.
```
let effect = ... in
```
it will fail with the effect syntax.

Reply:
Okay, so `effect` is now a keyword. I think I can just rename our uses of effect then.

Reply:
As you wish but this is basically OCaml 6.0 and the proposal hasn’t really been made yet. It only lives in opam-repository as `ocaml-variants.4.12.0+domains+effects`